exercises/list-ops/.meta/hints.md

```diff
@@ -1,3 +1,7 @@
 ## Hints
 
+In Java it's a best pracite to use instance methods over class methods. However, there are conditions in which it is absolutely appropriate for a function to be `static`. Since classes in Java are closed for modification (i.e. you cannot add members to a class outside its definition like you can in other languages like Ruby or JavaScript), you cannot add new behavior to the class, directly. What to do if you still want to define behavior for a given type? The idiomatic solution in this case is to write a utility method. 
```

Comment:
Should be "practice" :) And I don't think there should be a comma before "directly"? I'm not an expert on comma usage though so feel free to tell me if I'm wrong :P

Reply:
Also I think it should be "it's considered best practice" instead of "it's a best practice". What do you think?

Reply:
@FridaTveit you are right, I'll fixes those later today.
Thanks for pointing out 👍